spec/factories/documents.rb

```diff
@@ -0,0 +1,8 @@
+FactoryGirl.define do
+  factory :document do
```

Comment:
Consider adding Faker and making these values more realistic (and unique each time).